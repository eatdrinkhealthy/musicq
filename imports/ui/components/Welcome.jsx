import React from "react";
import { Link } from "react-router";

const Welcome = () => (
  <div>
    <h5>Welcome</h5>
    <Link to="/player">
      <span className="button">Player</span>
    </Link>
    <div className="row">
      <div className="five columns offset-by-three">
        <img src="images/music_note.png" alt="Music Note" className="u-max-full-width" />
      </div>
    </div>
  </div>
);

export default Welcome;
