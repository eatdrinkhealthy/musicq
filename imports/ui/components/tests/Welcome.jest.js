// @flow
/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */

import React from "react";
import renderer from "react-test-renderer";
import Welcome from "../Welcome";

describe("<Welcome />", function () {
  it("matches rendered snapshot", function () {
    const tree = renderer.create(<Welcome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

