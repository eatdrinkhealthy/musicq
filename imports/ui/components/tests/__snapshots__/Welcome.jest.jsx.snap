exports[`<Welcome /> matches rendered snapshot 1`] = `
<div>
  <h5>
    Welcome
  </h5>
  <a
    onClick={[Function]}
    style={Object {}}>
    <span
      className="button">
      Player
    </span>
  </a>
  <div
    className="row">
    <div
      className="five columns offset-by-three">
      <img
        alt="Music Note"
        className="u-max-full-width"
        src="images/music_note.png" />
    </div>
  </div>
</div>
`;
